feat: Determinar si la suma origina un numero par

Yerlis09 opened the pull request:
creacion de la funcionalidad que determina si la suma
de dos numeros originan un numero par, porqué se hacer
hace necesario contar con esta nueva caracteristica
para agilizar los calculos matematicos del proyecto

Tarea: #12

Yerlis09 committed:
feat: Determinar si la suma origina un numero par

creacion de la funcionalidad que determina si la suma
de dos numeros originan un numero par, porqué se hacer
hace necesario contar con esta nueva caracteristica
para agilizar los calculos matematicos del proyecto

Tarea: #12

oscles commented on 1_algoritmo.py:
```diff
@@ -151,3 +151,20 @@ def determinar_si_dos_digitos_son_iguales(numero = 0):
+def determinar_si_la_suma_origina_un_numero_par(numero1 = 0, numero2 = 0):
+    if not numero1 or not numero2:
+        return('ingrese valores validos, por favor')
+    elif (numero1 < 10 and numero1 > -10) or (numero2 < 10 and numero2 > -10):
```
Esta condicion puedes hacerla de la siguiente forma:
```python
if -10 < numero1 < 10 or -10 < numero2 < 10:
  # codigo

oscles reviewed (changes requested):
Buen trabajo, vamos mejorando pupilita :)

Yerlis09 committed:
fix: Corregir validaciones

Se corrigió las validaciones, porque
presentaban una inconsistencia innecesaria

Cambios realizados

Resuelve: #12

oscles commented:
Buenos días señorita cómo está aún no tenemos los cambios solicitados y mañana haremos despliegue en los servidores de producción.  Que tenga un hermoso día
